### DISCUSSION TITLE
New "argon" skin displays sliderhead judgements after the slider has finished.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20384] [Q&A] [answered]
The judgements for the sliders seem to be only drawn after the whole slider has been finished (i.e. the slider tail has been hit). You can see this behavior in the videos below

https://user-images.githubusercontent.com/102317848/191378505-f79b61fc-968b-4407-a38a-60cf90af594a.mp4

https://user-images.githubusercontent.com/102317848/191378477-64310665-f9de-421a-967c-f80c5ec1c2c0.mp4

This is more clear in longer sliders, where the slider would have finished by the time the hit judgement has faded out.
This seems to affect only the new "argon" skin, and not the triangles nor classic skins.

(Not sure if i should put this in issues or discussions, so I put it here. If it's supposed to be somewhere else, tell me so I can do better next time!)

**Comment by peppy** [comment, marked as answer]
This isn't even merged yet, so please leave feedback in the relevant PR thread rather than opening a discussion.

That said, the judgement *is* being displayed - it's just hidden by the slider. The judgement display is very as box and will improve over time.
